.gitignore

```diff
@@ -6,3 +6,4 @@ dist/
 build/
 ruly.egg-info
 test/.coverage
+.idea

```

Comment:
I didnt realize you would get notified about a PR on a fork.   Right now we are working on some updates to ruly and ruly-dmn so that they can work with the Camunda v5.0 DMN format.   Once we have something more generic we were going to make a PR suggestion to the main repo.
I will add this to be ignored on a global scale.

Reply:
Yeah, it's a PR to this repo so that's why I got notified, I would assume. Either way, I'm glad it's useful, I thought this might've been a Hacktoberfest PR initially :smile: